### ISSUE TITLE
3D printed case available?

### ISSUE THREAD

**Issue by bubbleytea** [good-enough-technology/Good-Enough_Air-Quality-Device_CO2-SCD4x_PM-SEN5x_CircuitPython#1]
Hi there! I'm trying to assemble one of these and I'm wondering if you had a link to the 3d printed case that the readme refers to?

**Comment by tyeth** [comment]
Not really, I'm not happy with the design as it traps too much heat. You can see all my things on Onshape as they are public.
I've been testing a laser cut version, but same issues, so I've moved onto a testing version with a fan for the esp32+display, to reduce the heating of the sensor. The 3d printable case will be a future addition. I may offer an open air version, and then a version with closed box and fan. I'll ping you and amend this issue as it develops.

I do have a better description / attempt at documenting, including a link to onshape with a search term populated.
See here:
https://adafruit-playground.com/u/tyeth/pages/wippersnapper-sensirion-sen55-particulate-voc-nox-sensor

**Comment by tyeth** [comment]
It's progressing. I have recently obtained a few skills as far as configuration options and variables go, so now you can switch between differing display+MCU combinations. I still need to add individual mounting options for each screen and mcu, and the fan holder is designed to have holes drilled for your threaded inserts. This is sadly more of a learning exercise rather than money making scheme, both in terms of CAD and thermal management :)
https://cad.onshape.com/documents/d5c926bd86ab27599638558c/w/c52b6c56eff5602eca7f6581/e/604c4a728e1f80c59f3f742a?configuration=List_rIyd8t0nQ9k2WB%3DPi5___Inky_Phat&renderMode=0&rightPanel=configPanel&uiState=65356e0e4d5b3463e086e72f!
![image](https://github.com/good-enough-technology/Good-Enough_Air-Quality-Device_CO2-SCD4x_PM-SEN5x_CircuitPython/assets/6692083/d547682a-6d59-4542-890c-d509dcb10d6f)

**Comment by tyeth** [comment]
Yet again the plot thickens, but it's mostly there now. Going to print a version not quite there to see how it feels.
https://cad.onshape.com/documents/85ac89b5536bfdf7b4f0608d/w/240a16609b8e6c62cde41201/e/5d7e2e1324a9fac14ad7416d
![image](https://github.com/good-enough-technology/Good-Enough_Air-Quality-Device_CO2-SCD4x_PM-SEN5x_CircuitPython/assets/6692083/5685c76c-67d0-41f9-9954-8435773fa601)

**Comment by tyeth** [comment]
![IMG_20231125_230838_642](https://github.com/good-enough-technology/Good-Enough_Air-Quality-Device_CO2-SCD4x_PM-SEN5x_CircuitPython/assets/6692083/cc8f0245-1a49-4dd9-97e4-9a08b056d6a5)
[Alpha3.zip](https://github.com/good-enough-technology/Good-Enough_Air-Quality-Device_CO2-SCD4x_PM-SEN5x_CircuitPython/files/13465445/Alpha3.zip)
This is the zip of the latest model, designed for a qualia + 3.2inch Bar touch display.

I think alpha4 will be the one 🥇
